readm.md

```diff
@@ -0,0 +1,29 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> My first Project with Microverse
+
+![screenshot](./img.png)
+
+A Repository to get used on installations,Github and GithubFlows
+
+## Built With
+
+- HTML
+- CSS
+## Authors
+
+👤 **Ntare Guy**
+
+- GitHub: [@41hulk](https://github.com/41hulk)
+- Twitter: [@ntare_guy](https://twitter.com/ntare_guy)
+- LinkedIn: [Ntare Guy](https://www.linkedin.com/in/ntare-guy/)
+
+## 🤝 Contributing
+
+My coding Partners and Awesome Friends, Ivan Silva and Mengstu Fentaw, I appreciate your help and let's Rock this🚀
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Kindly fix the `README.md` file name so that it can be easily accessed.
The file name you currently have is `readm.md`

Reply:
@Whoistolu all Solved